ingest/validation/validate_metadata.py

```diff
@@ -58,25 +57,27 @@ def validate_schema(json):
     """Check validity of metadata convention as JSON schema.
 
     :param schemafile: metadata convention JSON file
-    :return: jsonschema validator object using input convention
+    :return: if valid, jsonschema validator object using input convention
+                or returns None
     """
 
     try:
         jsonschema.Draft7Validator.check_schema(json)
         valid_schema = jsonschema.Draft7Validator(json)
         return valid_schema
     except jsonschema.SchemaError as e:
+        ### save this output as part of error output
         print('Input JSON is invalid as jsonschema;', e)
         return None
 
 
 def extract_numeric_headers(metadata):
-    """
+    """Find metadata headers of type numeric.
 
     REQUIRES if metadata_valid:
```

Comment:
updated to clarify "ASSUMES metadata.validate_format() == True"
If the file headers are improperly formatted, extract_numeric_headers may not be able to correctly identify the headers intended to be numeric. Does the updated statement need the context I'm providing in this comment?

Reply:
> Does the updated statement need the context I'm providing in this comment?

Yes, it would help!  I know I'd benefit from having your prose comment in the code.